stock_move_source_relocate/models/stock_move.py

```diff
@@ -63,3 +78,7 @@ def _apply_source_relocate_rule(self, relocation, reserved_availability, roundin
         new_move.location_id = relocation.relocate_location_id
         self._action_assign()
         return new_move
+
+    def _after_apply_source_relocate_rule(self):
+        # Hook for stock_dynamic_routing
```

Comment:
Not sure we need to mention this module in `stock_move_source_relocate`? It's the job of the glue module (or anything else) to use this hook.